### ISSUE TITLE
Images in Gmail email don't display

### ISSUE THREAD

**Issue by tonydm** [Alex313031/thorium#425]
**System Details**
OS: MX Linux 23.1 KDE (bookworm)
Thorium Version: 117.0.5938.157

**Problem**
Cannot view external images in email.

The option in the email being viewed:
![Selection_058](https://github.com/Alex313031/thorium/assets/1261605/7c1ddd12-f409-4ad4-bc34-60aa4071bd55)
However, when I click "Display images below", the option goes away but no images are displayed.

I have gone to Settings->See all settings
![Selection_059](https://github.com/Alex313031/thorium/assets/1261605/b3dc2892-c03f-4876-8030-fe68bf3b37e0)
and ensured General/Images is set to "Ask before displaying external images".
However, even setting to "Always display external images", email does not display images.  If I change setting back to "Ask before displaying external images" and view the email again, the option to "Display images below" is shown.  But clicking the "Display images below" only removes the option and no images in the email are displayed.

Viewing the same email in Chromium or Chrome, works as intended.  It shows the option to display images below or always display external images relative to the Settings/General/Images "Ask before displaying external images".

I'm not sure if this is a bug or a setting I am overlooking.

Thank you very much for your project and efforts to support!

**Comment by gz83** [comment]
I can't reproduce the problem you encountered, try in incognito mode to see if it works normally

**Comment by tonydm** [comment]
Thanks @Alex313031 - Tested as requested in Incognito Mode... works as expected.  So, 64mil$?, why not in std mode and what to do about it?  Seems like a silly thing, but if I can't view images in my emails then, sadly, it's kind of a deal breaker.  I'm not a fan of Thunderbird... would welcome an Email Desktop App but shed TB years ago and haven't seen any other options that were unadulterated.

**Comment by gz83** [comment]
If it works normally in incognito mode, it may be caused by some extensions. It is recommended to troubleshoot each extension one by one.

**Comment by tonydm** [comment]
Hmmm, found culprit, I think...

I've used [AdBlock](https://chromewebstore.google.com/detail/adblock-%E2%80%94-best-ad-blocker/gighmmpiobklfepjocnamgkkbiglidom?pli=1) for years w/o issue.  So not on my radar.  But that's the first Ext I tested at your suggestion.  Disabled the ext and it seemed to resolve.  But, set and saved my Settings/General/Images setting to "Always display" and back again to "Always ask" to reset the email display.  Re-enabled the AdBlock ext and tested again.  But... this time, clicking the "Display images below" worked as expected with the AdBlock ext enabled.  So don't know what to think.   But for now it seems to work and I'll keep an eye on it.  Thanks again for the super fast response!!!
